src/Stamp.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { space } from 'styled-system'
+import theme from './theme'
+
+const type = props => {
```

Comment:
NB: I'd suggest naming this something more explicit like `getColorCombination` – these sorts of color combinations and the ones in Badge, Button, etc. would be good to abstract out to the theme at some point